### ISSUE TITLE
No indication of open source license

### ISSUE THREAD

**Issue by philomory** [jerakia/python-jerakia#12]
I can't help but notice that the software in this repository has no assigned open source license that I can tell (nothing in the README, no LICENSE file, etc). Other software from the Jerakia organization (such as jerakia/jerakia) is under the Apache 2.0 license. Is the intent that this package is licensed the same way, or did you have something else in mind?

**Comment by hhenkel** [comment]
@beddari @crayfishx  Any thoughts on this? Guess it would make sense to use the same license as for the rest of the code?

**Comment by hhenkel** [comment]
Sorry @jonandernovella forgot to mention you as well.

**Comment by beddari** [comment]
Yeah I think that assuming the same license here is just fine, so we can go ahead and add it.
